requirements.txt

```
@@ -1,5 +1,5 @@
 --extra-index-url https://pypi.anaconda.org/OpenEye/simple
-OpenEye-toolkits >= 2024.1.1
+OpenEye-toolkits >= 2022.1
```

This looks OK, but just note that the use of `>=` will probably install the latest 2024 version anyway. I'd also put a comment here, e.g.:

```suggestion
OpenEye-toolkits >= 2022.1  # First version compatible with Python 3.10
```

---

@piehld I added the comment.

> just note that the use of >= will probably install the latest 2024 version anyway.

This change allows dependency resolution with these constraints:

```
rcsb.utils.chem >= 0.82
OpenEye-toolkits == 2022.1
```

Otherwise, we have this conflict:

```
rcsb.utils.chem >= 0.82
\---> OpenEye-toolkits >= 2024.1.1
OpenEye-toolkits == 2022.1
```